Resources/Prototypes/Roles/Jobs/Command/head_of_personnel.yml

```diff
@@ -32,31 +33,32 @@
   - Janitor
   - Theatre
   - Kitchen
-#  - Chapel
+  # - Chapel # Parkstation-Epistemics
   - Hydroponics
```

Comment:
HoP should not have any of these unfortunately, as they're part of Service. Worry not, I've had some ideas to alleviate HoP's lack of Access issues in the future, but this PR should simply remove all the ones that don't apply.

Missed this in the review, my bad :P

Reply:
Wait what, I though CCS is the service head and HoP is the general low AA. Atleast I though that was the parkstation thing. Well okay.